src/transactions/RestoreFootprintOpFrame.cpp

```diff
@@ -163,6 +163,40 @@ class RestoreFootprintApplyHelper : virtual public LedgerAccessHelper
                             lk, entry, ledgerSeq, getLedgerVersion());
                 }
 
+                if (mApp.getProtocol23CorruptionEventReconciler())
```

Comment:
Do we actually have any restoration ops that corrupted the data? I'm pretty sure we don't, as at some point we had a bug in invariant validation in restore op, and invariant has still passed. Given that this is a significant chunk of code, it would be nice to not maintain it.

Reply:
Yeah I mentioned this in a comment above. I didn't see any restore op cases, so it can be removed.

Reply:
Done